### PR TITLE
Remove version check for equivalency operator.

### PR DESCRIPTION
**Pull request by losborne**
Affects Code, Value Set, and Concept equivalency.
Equivalency should no longer return false if the only difference is in version.

Modified Equivalent class to use the equivalent function in comparisn.js rather
than a different but similar expression.

The only tests that failed due to this change were expected (tests that were
looking for different versions). I updated those tests to expect equivalence if
the only difference is in version.

JIRA Ticket: https://jira.mitre.org/browse/BONNIE-1102
Corresponding cql_qdm_patientapi PR: https://github.com/projecttacoma/cql_qdm_patientapi/pull/84

### CODE REVIEW

**Comment by losborne** [comment]
Regression script was run and passed -- all results equal.

**Comment by cmoesel** [comment]
Assuming we need to wait for @jbradl11 review before merging, but let me know if otherwise.

File: Src/coffeescript/cql-execution/test/util/comparison.coffee [hunk 1]
```diff
@@ -113,3 +113,6 @@ describe 'equivalent', ->
     equivalent('123', '123').should.be.true()
     equivalent(123, 123).should.be.true()
     equivalent('123', new Code('123', 'test', '2016')).should.be.false()
+
+  it 'should consider codes with different versions to be equivalent', ->
```
Comment (hossenlopp) [inline Src/coffeescript/cql-execution/test/util/comparison.coffee]: I think a test should be added to check equivalence between code and concept/code lists.

Comment (losborne) [reply Src/coffeescript/cql-execution/test/util/comparison.coffee]: Added some below.

**Comment by losborne** [comment]
Should be fixed now--missed one name change when I went to make the lines a bit shorter.

**Pull request merged**